### PR TITLE
add pyflakes linter

### PR DESCRIPTION
**Pull request by Akuli**
This is the same tool that Porcupine uses internally for displaying red and yellow underlines in code. It warns about unnecessary imports, potentially undefined variables, and a few other things

I switched the order of the if,else statement in `recv_loop()` to make pyflakes happy. It "reads" the code from top to bottom, and assumes everything is defined before (i.e. above) it is used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,10 +41,11 @@ Some handy resources for developing IRC-related programs:
...
 
-Any pull request will automatically be checked for proper format by `black`, `mypy` (--strict mode), as well as by `pytest`
+Any pull request will automatically be checked for proper format by `black`, `pyflakes`, `mypy` (--strict mode), as well as by `pytest`
 
 To make sure the PR will pass these checks, please use the following commands before pushing:
 
 - `black *.py tests/*.py`
 - `mypy --strict *.py`
 - `python3 -m pytest tests/*.py`
+- `python3 -m pyflakes *.py`
```
Comment (Akuli) [inline README.md:51]:
```suggestion
- `python3 -m pyflakes *.py tests/`
```

Comment (ThePhilgrim) [reply README.md]: @Akuli Doesn't it have to be `python3 -m pyflakes *.py /tests/*.py`?

**Comment by ThePhilgrim** [comment]
@Akuli I'm on IRC if you wanna come early today

File: README.md (hunk 1 above)
Comment (Akuli) [reply README.md]: - Definitely not `/tests/`, as that is an absolute path that likely doesn't exist, and outside the `mantatail` repo.
- It accepts folders or files, so `tests/*.py` would work too.

**Pull request merged**